### PR TITLE
fix: crash when startup if the milvus volume is on-operation concurrently

### PR DESCRIPTION
**Pull request by chyezh**
issue: #37311
pr: #37312

### CODE REVIEW

**Comment by chyezh** [comment]
rerun go-sdk

**Comment by chyezh** [comment]
rerun go-sdk

File: internal/core/src/storage/LocalChunkManager.cpp [hunk 1]
```diff
@@ -232,7 +233,17 @@ LocalChunkManager::GetSizeOfDir(const std::string& dir) {
...
+            auto file_size = boost::filesystem::file_size(it->path(), ec);
+            if (ec) {
+                // The file may be removed concurrently by other threads.
+                // So the file size cannot be obtained, just ignore it.
+                LOG_INFO("size of file {} cannot be obtained with error: {}",
+                         it->path().string(),
+                         ec.message());
+                continue;
+            }
+            total_file_size += file_size;
```
Comment (liliu-z) [inline internal/core/src/storage/LocalChunkManager.cpp:246]: if error, will the `file_size` be 0?

Comment (chyezh) [reply internal/core/src/storage/LocalChunkManager.cpp:246]: if error, the loop is continued, file_size is ignored.

**Pull request merged**